packages/types/tsconfig.build.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.build.json",
+  "compilerOptions": {
+    "outDir": "./dist"
```

Comment:
This should output into a ./types folder, like the other packages IMO. This current solution confuses my IDE a bit (for some reason it thinks that code in ./dist is not autogenerated

Reply:
Fixed